spec/components/all_components_spec.rb

```diff
@@ -22,7 +22,7 @@
       end
 
       it "has the correct class in the ERB template",
-        skip: component_name.in?(%w[step_by_step_nav_related step_by_step_nav_header step_by_step_nav previous_and_next_navigation]),
+        skip: component_name.in?(%w[step_by_step_nav_related step_by_step_nav_header step_by_step_nav previous_and_next_navigation checkboxes]),
```

Comment:
We shouldn't need to skip this one.

Reply:
Yes, how do we do this? I wasn't familiar with this test. It looks like it's checking to see there's a correctly formatted class name for the component in the template, but in this case we're generating lists of classes and passing them into the template from the helper. I could move some of that code into the template, but it seems unnecessary? I'm not sure why this test is useful - it looks like I could put the class name in a comment and it would pass.

Reply:
Well.. I wasn't familiar either, but I guess it's our way of making sure we're not missing these 'tag'/'selector' classes when we create a new component or update classes on another. Ideally we would check the generated markup instead of the ERB file, but until we update the tests to do that could we add `# cb_helper.css_classes generates "gem-c-checkboxes"`? instead of listing this as an exception. It's not great either way, so up to you.

What bothers me is that we're branching more and more the checkboxes from radios, when in theory they should be very similar.